onevizion/trackor.py

```diff
@@ -487,23 +487,38 @@ def get_filename_from_cd(cd):
 	def UploadFile(self, trackorId, fieldName, fileName, newFileName=None):
 		""" Get a File from a particular Trackor record's particular Configured field
 
-			trackorID: the system ID for the particular Trackor record that this is being assigned to.
+			trackorId: the system ID for the particular Trackor record that this is being assigned to.
 			fieldName: should be the Configured Field Name, not the Label.
 			fileName: path and file name to file you want to upload
 			newFileName: Optional, rename file when uploading.
 		"""
 
+		FilePath = fileName
+		FileName = newFileName if newFileName else os.path.basename(FilePath)
+		BinaryStream = open(FilePath, 'rb')
+
+		Message("FilePath: {FilePath}".format(FilePath=FilePath),2)
+
+		self.UploadFileByFileContents(trackorId=trackorId, fieldName=fieldName, fileName=FileName, fileContents=BinaryStream)
+
+
+	def UploadFileByFileContents(self, trackorId, fieldName, fileName, fileContents):
+		""" Get a File from a particular Trackor record's particular Configured field
```

Comment:
Why the comment says `Get a File`? Shouldn't is say `Upload a file to a ...`

Reply:
Fixed